server-code/webhook.pl

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/perl
+# !!!REMINDER!!!: I have removed the secret key, you must
```

Comment:
Can we use a similar tactic as the SQE_RESTRICTED.PM? i.e., use a gitignored file that contains the Key. And then you can simply include it in and supply that value?

Reply:
That is a good idea, but I think that is not terribly necessary.  This file is only for people running their own production server and their own GitHub repository that they have setup to use webhooks.  I set this up on the server and don't really rely on the GitHub project for maintaining it.  Let me give it some thought though.  There may be some use for something in .gitignore, but I don't actually even keep a version of the Scrollery-website package on the server.